include/swift/AST/Types.h

```diff
@@ -4431,6 +4439,22 @@ class SILFunctionType final : public TypeBase, public llvm::FoldingSetNode,
 
   const clang::FunctionType *getClangFunctionType() const;
 
+  /// Given that `this` is a `@differentiable` or `@differentiable(linear)`
+  /// function type, returns an `IndexSubset` corresponding to the
+  /// differentiability/linearity parameters (e.g. all parameters except the
+  //// `@noDerivative` ones).
```

Comment:
extra "/"

Reply:
Thanks! Fixed in https://github.com/apple/swift/pull/29674/commits/d0b77eb98c3d8f0bbd15bc6901b906bcace4cb79.